### PR TITLE
Remove unnecessary double quotes

### PR DESCRIPTION
**Pull request by BanzaiMan**
These double quotes should not be in the file content.

### CODE REVIEW

**Comment by lancehalvorsen** [comment]
Hi @BanzaiMan,

Thank you for taking the time to put together this PR!

Since both of these examples are inside of Elixir code blocks, they need to be valid Elixir code. The way to do that is to make them Elixir strings, so the double quotes are necessary.

Thanks again.

**Comment by BanzaiMan** [comment]
If that is the case, these snippets should not be marked as Elixir code segments.

File: D_controllers.md [hunk 1]
```diff
@@ -393,7 +393,7 @@ end
 And let's add a bit to our text template.
 
 ```elixir
-"OMG, this is actually some text." <%= @message %>
+OMG, this is actually some text. <%= @message %>
 ```
 
 Now if we go to `http://localhost:4000/?_format=text&message=CrazyTown`, we will see "OMG, this is actually some text. CrazyTown"
```
Comment (BanzaiMan) [inline D_controllers.md]: This output is incorrect. The correct output would be:

```
"OMG, this is actually some text." CrazyTown
```

**Comment by lancehalvorsen** [comment]
Before the year is out, I must say that you are correct! Thank you for persevering, and I very much appreciate this PR. Happy New Year! :tada:

**Comment by BanzaiMan** [comment]
@lancehalvorsen No worries. Thanks for merging! Happy New Year!

**Pull request merged**